### PR TITLE
changes to enable concurrent runs - Close #300

### LINKED ISSUE #300: Allow concurrent Jenkins runs
### Expected behaviour
Runs can be done concurrently
### Actual behaviour
Runs are serial

### PR DESCRIPTION
**Pull request by Nazgolze**
It needs a little tidying up (especially that line where I start explorer and I'm specifying the redis port twice), but so far, it "works on my box"

Closes #300

### CODE REVIEW

**Comment by Nazgolze** [comment]
Talked it over with Isabella, I feel pretty good about this working. Going 6 at a time causes issues, especially with the getPriceTicker call since that's rate limited. 3-4 at a time works pretty well

File: Jenkinsfile [hunk 1]
```diff
@@ -1,134 +1,198 @@
...
 
     stage ('Build Dependencies') {
       try {
-        sh '''#!/bin/bash
+        sh '''
 
         # Install Deps
         npm install
+        ./node_modules/protractor/bin/webdriver-manager update
+        wget https://downloads.liskwallet.net/lisk/redis/redis-3.2.9-Linux-x86_64.tar.gz
```
Comment (Isabello) [inline Jenkinsfile]: Please just upload the binaries to the jenkins machine instead of dowloading every time.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,134 +1,198 @@
...
-        cp test/config_lisk.json ~/lisk-test/config.json
-        bash ~/lisk-test/lisk.sh rebuild -f ~/lisk-test/blockchain_explorer.db.gz
+
+        sh '''
+        N=${EXECUTOR_NUMBER:-0}
+        # work around core bug: config.json gets overwritten; use backup
+        cp test/config_lisk.json ~/lisk-test/config_$N.json
+        cd ~/lisk-test
+        # change core port
+        sed -i -r -e "s/^(.*ort\\":) 4000,/\\1 400$N,/" config_$N.json
```
Comment (Isabello) [inline Jenkinsfile]: you could probably use JQ instead of sed to do this.

Comment (Isabello) [reply Jenkinsfile]: We should  also leave core permanently running and remove alot of these steps, it doesnt hurt that more than one node would query it for testing.

**Comment by Isabello** [comment]
Consider using database number instead of seperate redis instances

**Pull request merged**